cpp/cmake_modules/SetupCxxFlags.cmake

```diff
@@ -618,6 +618,15 @@ endif()
 
 message(STATUS "Build Type: ${CMAKE_BUILD_TYPE}")
 
+# Workaround https://github.com/xtensor-stack/xsimd/issues/573
```

Comment:
To workaround xsimd build error if gcc -O0 on arm64. Disable xsimd if gcc & neon & debug build.
Details at https://github.com/xtensor-stack/xsimd/issues/573. This looks to me a compiler defect.

Reply:
Cool, let's wait for it then?

Reply:
Yes.